sonar-plugin/sonar-javascript-plugin/src/main/java/org/sonar/plugins/javascript/analysis/AnalysisProcessor.java

```diff
@@ -185,7 +185,12 @@ private void saveIssues(List<Issue> issues) {
         file,
         issue.line()
       );
-      saveIssue(issue);
+      try {
+        saveIssue(issue);
+      } catch (RuntimeException e) {
+        LOG.warn("Failed to save issue in {} at line {}", file.uri(), issue.line());
+        LOG.warn("Reason: {}", e.getMessage());
```

Comment:
I would suggest to use the signature with `Throwable`, it will also include the stacktrace in the log
```suggestion
        LOG.warn("Exception cause", e);
```